infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/AsyncStorageProvisioner.java

```diff
@@ -333,19 +336,37 @@ private void createAsyncStoragePodIfNotExist(
     PodSpec podSpec =
         podSpecBuilder.withContainers(container).withVolumes(storageVolume, sshKeyVolume).build();
 
-    Pod pod =
-        new PodBuilder()
-            .withApiVersion("v1")
-            .withKind("Pod")
-            .withNewMetadata()
-            .withName(ASYNC_STORAGE)
+    ObjectMetaBuilder metaBuilder = new ObjectMetaBuilder();
+    ObjectMeta meta =
+        metaBuilder
+            .withLabels(
+                of(
+                    "app",
```

Comment:
Why do we label this as belonging to a different application than che? I thought `app=che` is the "overarching" label that groups everything that Che is composed of..

Reply:
ok